src/understanding-workflow/concepts.md

```diff
@@ -108,11 +108,11 @@ Note that during the deploy, your application will be running in a mixed mode.
 
 Workflow treats all persistent services such as databases, caches, storage,
 messaging systems, and other [backing services][] as resources managed
-separtely from your application. This philosophy aligns with Twelve-Factor
+separately from your application. This philosophy aligns with Twelve-Factor
 best practices.
 
-Applications are attached to backing services using [environment variables][].
-Because applications are decoupled from backing services, apps are free to
+Applications attach to backing services using [environment variables][].
```

Comment:
Is this referring to the standard k8s service discovery env vars? If so, maybe a good idea to link to the [Kubernetes services docs](http://kubernetes.io/docs/user-guide/services/)